---
title: "&#39; Catch &#39; dışında görünemez bir &#39; Try &#39; deyimi"
ms.date: 07/20/2015
ms.prod: .net
ms.technology: devlang-visual-basic
ms.topic: article
f1_keywords:
- bc30380
- vbc30380
helpviewer_keywords: BC30380
ms.assetid: 73ce950d-881f-4532-8024-40a4930abd32
caps.latest.revision: "7"
author: dotnet-bot
ms.author: dotnetcontent
ms.openlocfilehash: 62502b364e8609ba66e59507554029e67b563269
ms.sourcegitcommit: 4f3fef493080a43e70e951223894768d36ce430a
ms.translationtype: MT
ms.contentlocale: tr-TR
ms.lasthandoff: 11/21/2017
---
# <a name="39catch39-cannot-appear-outside-a-39try39-statement"></a>&#39; Catch &#39; dışında görünemez bir &#39; Try &#39; deyimi
`Catch`içinde yer almalıdır bir `Try...Catch...Finally` deyimi bloğu. Ya da bir gereksiz sahip `Catch` deyiminde, `Try` bloğu veya `Catch` deyimi görünüyor ilgili sınırları dışında `Try` bloğu.  
  
 **Hata Kimliği:** BC30380  
  
## <a name="to-correct-this-error"></a>Bu hatayı düzeltmek için  
  
1.  Silme `Catch` gereksiz veya içine koyun deyimi bir `Try...Catch...Finally` deyimi bloğu.  
  
## <a name="see-also"></a>Ayrıca Bkz.  
 [Try... Catch... Finally ifadesi](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)  
 [Yapılandırılmış özel durum için Visual Basic genel bakış işleme](http://msdn.microsoft.com/en-us/bb81af80-a735-4873-9711-6151a48e418a)
